src/app/cards/cards.component.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    AfterViewInit,
+    Component,
+    ElementRef,
+    inject,
+    Input,
+    QueryList,
+    ViewChildren,
+} from '@angular/core';
+import { CommonModule, NgOptimizedImage } from '@angular/common';
+import { ReactiveFormsModule } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { HighlightModule } from 'ngx-highlightjs';
+import { NgxLoaderIndicatorDirective } from 'ngx-loader-indicator';
+import { ICard } from './cards.interface';
+import { TrackByService } from '@libraries/track-by/track-by.service';
+import { ScrollService } from '@open-source/scroll/scroll.service';
+import { ColorPipe } from '@open-source/color/color.pipe';
+import { AssetPipe } from '@libraries/asset/asset.pipe';
+import { CustomLoaderComponent } from '../custom-loader/custom-loader.component';
+import { FormComponent } from '../shared/form/form.component';
+import { CardType } from './cards.enum';
+import { InputComponent } from '@libraries/input/input.component';
+import { OpenSourcePath } from '@open-source/path/open-source.path';
+
+@Component({
+    selector: 'jsdaddy-open-source-cards',
+    standalone: true,
+    imports: [
+        CommonModule,
+        NgOptimizedImage,
+        HighlightModule,
+        ReactiveFormsModule,
+        NgxLoaderIndicatorDirective,
+        ColorPipe,
+        AssetPipe,
+        CustomLoaderComponent,
+        FormComponent,
+        InputComponent,
+    ],
+    providers: [ScrollService],
+    templateUrl: './cards.component.html',
+    styleUrls: ['./cards.component.scss'],
+})
+export class CardsComponent implements AfterViewInit {
```

Comment:
as i see we have it in the all open source. is it make sense to make it reusable?

Reply:
interfaces are different